src/UI/Component/Glyph/Glyph.php

```diff
@@ -60,4 +62,20 @@ public function getCounters();
 	 * @return	Glyph
 	 */
 	public function withCounter(Counter $counter);
+
+
+	/**
+	 * Returns whether the Glyph is highlighed
+
+	 * @return bool
+	 */
+	public function isHighlighted();
+
+	/**
+	 * Highlights a Glyph (sets the highlighted class)
+	 *
+	 * @param bool|true $highlighted
+	 * @return mixed
+	 */
+	public function highlighted($highlighted = true);
```

Comment:
Oke, thx, thought about this option, thought it sounds a little funny. Do not have a strong opinion though. withHighlight is fine with me (+1 for consistency).